pkg/openshift/certgen/unstable/templates/master/tmp/ansible/azure-local-master-inventory.yml

```diff
@@ -65,7 +65,6 @@ localmaster:
 
     ansible_service_broker_install: True
     openshift_service_catalog_image_prefix: "IMAGE_PREFIX/IMAGE_TYPE-"
-    openshift_service_catalog_image_version: 'vVERSION'
```

Comment:
According to this, https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_service_catalog/vars/openshift-enterprise.yml, it appears that the variable will take on the `opnehsift_image_tag` which will be set to the version when doing ocp.  This is equivalent to setting it to `vVERSION` for us.

When doing origin they have defaulted version to be this, https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_service_catalog/vars/default_images.yml#L3.

Reply:
After our IRC discussion I was able to remove `oreg_url_master` and redeploy successfully.

```
INFO[0013] Starting ARM Deployment (kwoodsoncluster1-1987505677). This will take some time... 
INFO[0758] Finished ARM Deployment (kwoodsoncluster1-1987505677). Succeeded 
```